src/types/models/carSales.ts

```diff
@@ -28,3 +28,8 @@ export interface CarSales {
   id: number
   listing: CarSalesListing
 }
+
+export interface CarSaleRejection {
+  comment?: string
+  reason: "car-not-sold" | "wrong-buyer"
+}
```

Comment:
I think we need to add `CarSalesRejection` into `CarSales` as `rejection: CarSalesRejection` or maybe I'm missing something there.

Reply:
U right missed that it was missing in the data response as well, added it